script.js

```diff
@@ -0,0 +1,124 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.navi-menu');
+const logo = document.querySelector('.logo');
+const nav = document.querySelector('.nav');
+function mobileMenu() {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  logo.classList.toggle('active');
+  nav.classList.toggle('active');
+}
+hamburger.addEventListener('click', mobileMenu);
+const naviLink = document.querySelectorAll('.navi-link');
+
+function closeMenu() {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  logo.classList.remove('active');
+  nav.classList.remove('active');
+}
+
+naviLink.forEach((n) => n.addEventListener('click', closeMenu));
+
+// pop-up
+const projectDatas = [
+  {
```

Comment:
- [ ] Kindly align the buttons to be in the same line as the bottom of the pop-up image to the left as is in the original design.

Your design
![image](https://user-images.githubusercontent.com/81932478/208919245-af9ec6e8-cdf8-4880-89fe-0d1942f8264c.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/208919325-c18f244f-9c04-4abc-b306-a11df4b70049.png)